src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt

```diff
@@ -99,11 +97,12 @@ class AlertsDpsApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
-  fun stubMigrateAlert(
```

Comment:
some of these mocks no longer referenced so deleted (but makes diff look a bit odd)